torch/fx/node.py

```diff
@@ -1,33 +1,52 @@
 # Nodes represent a definition of a value in our graph of operators.
-from typing import TYPE_CHECKING, Union, Callable, Any, Tuple, List, Optional, Dict, Set
-from ._compatibility import compatibility
-from .immutable_collections import immutable_dict, immutable_list
-import torch
 import builtins
 import types
 import warnings
-from torch.fx.operator_schemas import normalize_function, normalize_module, ArgsKwargsPair
+from typing import TYPE_CHECKING, Union, Callable, Any, Tuple, List, Optional, Dict, Set
+
+import torch
+from torch.fx.operator_schemas import (
+    normalize_function,
+    normalize_module,
+    ArgsKwargsPair,
+)
+from ._compatibility import compatibility
+from .immutable_collections import immutable_dict, immutable_list
```

Comment:
If these changes are being done due to an automatic IDE formatting plugin, please disable that plugin and revert these changes. Extraneous formatting changes mess up the blame information in the source files and can make further development more difficult

Reply:
I have reverted the formatting changes